packages/redux/src/users/returns/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,179 @@
+        },
+      };
+
+      const newState = reducer(previousState, {
+        payload: newResult,
+        type: actionTypes.FETCH_USER_RETURNS_SUCCESS,
+      });
+
+      expect(newState.result).toEqual(newResult);
+    });
+
+    it('should handle other actions by returning the previous state', () => {
+      const state = {
+        result: mockNormalizedUserReturnsResponse.result,
+        error: null,
+        isLoading: false,
+      };
+
+      const randomActionWithResult = {
+        type: 'random',
+        result: [{ id: '222' }],
+      };
+
+      expect(reducer(state, randomActionWithResult).result).toBe(state.result);
+    });
+  });
+
+  describe('error() reducer', () => {
+    it('should return the initial state', () => {
+      const state = reducer(undefined, { type: 'INIT' }).error;
+
+      expect(state).toBe(initialState.error);
+      expect(state).toBeNull();
+    });
+
+    // Error value on FAILURE
+    it.each([actionTypes.FETCH_USER_RETURNS_FAILURE])(
+      'should handle %s action type',
+      actionType => {
+        const error = 'foo';
+        const mockState = { ...initialState, error: null };
+        const reducerResult = reducer(mockState, {
+          payload: { error },
+          type: actionType,
+        });
+
+        expect(reducerResult.error).toBe(error);
+      },
+    );
+
+    // Error value on REQUEST
+    it.each([actionTypes.FETCH_USER_RETURNS_REQUEST])(
+      'should handle %s action type',
+      actionType => {
+        const previousState = {
+          ...initialState,
+          error: toBlackoutError(new Error('foo')),
+        };
+        const reducerResult = reducer(previousState, {
+          type: actionType,
+        });
+
+        expect(reducerResult.error).toBe(initialState.error);
+      },
+    );
+
+    it('should handle other actions by returning the previous state', () => {
+      const state = {
+        ...initialState,
+        error: new Error('foo') as BlackoutError,
+      };
+
+      const randomActionWithError = {
+        type: 'random',
+        payload: { error: new Error('dummy error') },
+      };
+
+      expect(reducer(state, randomActionWithError).error).toBe(state.error);
+    });
+  });
+
+  describe('isLoading() reducer', () => {
+    it('should return the initial state', () => {
+      const newState = reducer(undefined, { type: 'INIT' }).isLoading;
+
+      expect(newState).toBe(initialState.isLoading);
+      expect(newState).toBe(false);
+    });
+
+    // Loading status on REQUEST
+    it.each([actionTypes.FETCH_USER_RETURNS_REQUEST])(
+      'should handle %s action type',
+      actionType => {
+        expect(
+          reducer(undefined, {
+            type: actionType,
+          }).isLoading,
+        ).toBe(true);
+      },
+    );
+
+    // Loading status on SUCCESS
+    it.each([actionTypes.FETCH_USER_RETURNS_SUCCESS])(
+      'should handle %s action type',
+      actionType => {
+        expect(
+          reducer(undefined, {
```

Comment:
This case should be tested with a state whose `isLoading` value is true (the same for the fetch user returns failure case below).